packages/core/strapi/lib/core/loaders/plugins/get-enabled-plugins.js

```diff
@@ -9,6 +9,8 @@ const loadConfigFile = require('../../app-configuration/load-config-file');
 
 const isStrapiPlugin = info => get('strapi.kind', info) === 'plugin';
 
+const isPath = path => /^(\.\/|\.\.\/|\/|\.\\|\.\.\\|\\)/.test(path);
```

Comment:
You could simplify this by doing: 
```js
const isPath = path => /^(\.{0,2}(\/|\\))/.test(path);
```

You can also use `path.sep` to get the separator used by the server: 
```js
const isPath = path => new RegExp('^(.{0,2}\\' + path.sep + ')').test(path);
```